### PR TITLE
Fix #190

### LINKED ISSUE #190: Documentation contains a bit misleading...
Hi,

Thanks about a great plugin. But I think documentation contains a bit misleading.
`wildcardAny` and `wildcardOne` value is not changeable in `Model::$filterArgs`.

For example:
https://github.com/CakeDC/search/blob/master/Docs/Documentation/Examples.md#advanced-usage

Please let me know if I wrong. :)

### PR DESCRIPTION
**Pull request by tsmsogn**
Patch for #190

### CODE REVIEW

**Comment by tsmsogn** [comment]
It is as you say, changing wildcard per field can easily cause confusion. I have not been encountered such need too.

File: Model/Behavior/SearchableBehavior.php [hunk 1]
```diff
@@ -307,7 +307,7 @@ protected function _addCondLike(Model $Model, &$conditions, $data, $field, $like
 				$field['after'] = '%';
 			}
 
-			$options = $this->settings[$Model->alias];
+			$options = array_merge($this->settings[$Model->alias], $field);
```
Comment (dereuromark) [inline Model/Behavior/SearchableBehavior.php]: String key based options should be merged with `+` by the way.

Comment (tsmsogn) [inline Model/Behavior/SearchableBehavior.php]: Thanks advice.

**Comment by tsmsogn** [comment]
Though written in docs, I came to think it is not need.

**Comment by omnuvito** [comment]
Hi @tsmsogn, please send it to develop instead and check failing unit tests https://travis-ci.org/CakeDC/search/builds/67933361.

**Pull request closed without merging**